app/serializers/question_serializer.rb

```diff
@@ -39,8 +39,6 @@ def options
         object.dropdown_options
       when 'RangeOption'
         object.range_options
-      when 'file'
```

Comment:
Does this mean we might not need the `FileOption` model at all?

Reply:
It means that if it's a `FileOption` or `FreeOption`, there aren't really any options to choose from, so we'll just return the default empty array